### DISCUSSION TITLE
How to use geometric multigrid preprocessing(mg)？

### DISCUSSION THREAD

**Discussion by Moyiqi** [idaholab/moose#23993] [Q&A Getting Started] [answered]
Hello, everyone! May I ask a question?
When I use mg, it seems that geometric information cannot be applied, only one layer can be applied. Could you please tell me whether multiple layers of mg can be used？
Thanks!
Lily

**Comment by GiudGiud** [comment]
Hello

How are you using geometric multigrid from MOOSE?

Guillaume

**Reply by Moyiqi** [reply]
I do this by using the option -pc_type mg.
Lily

**Comment by lindsayad** [comment, marked as answer]
We do not have handles for true geometric multigrid in MOOSE at this time, although I know it is on @roystgnr's wish-list. You can see some geometric multigrid at the libMesh level in https://github.com/libMesh/libmesh/pull/2047

**Reply by Moyiqi** [reply]
Thanks a lot!
Lily
